CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## Release History
 
+### 1.9.0 (2023-06-19)
+#### New Features
+* Updated `azure-cosmos` version to 4.46.0.
+
+#### Breaking Changes
+* This change will stop updating the lease container continuation token based on kafka offset when using the config `connect.cosmos.offset.useLatest`. [PR 516](https://github.com/microsoft/kafka-connect-cosmosdb/pull/516)
+
+When set to `false`, if the lease container exists, connector will now process the changes from the current continuationToken in `leaseContainer`, if the lease container does not exist, then it will start process the changes from beginning.
```

Comment:
NIT: then it will start processing the changes from beginning.

Reply:
will resolve in a different PR